streams/src/main/java/org/apache/kafka/streams/state/internals/ThreadCache.java

```diff
@@ -196,7 +197,12 @@ public MemoryLRUCacheBytesIterator all(final String namespace) {
         return new MemoryLRUCacheBytesIterator(cache.allKeys(), cache);
     }
 
-
+    public NavigableSet<Bytes> keySet(final String namespace) {
+        final NamedCache cache = getCache(namespace);
+        if (cache == null) return null;
```

Comment:
return an `emptySet` rather than null?

Reply:
I am not sure if this is neccesary. Since {{NavigableSet}} is a interface, I will need to return a class which implements it, which means an extra import for this sole purpose.

Reply:
IMO it would be better to avoid returning `null` when you are returning a collection type